<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>Transformation of ylic133_5 with the PCR products from yll137 and yll140  | Experimental-Journal-PhD</title>
    <meta name="description" content="">
    
    
    <link rel="preload" href="/Experimental-journal-deploy/assets/css/0.styles.1dc62a3a.css" as="style"><link rel="preload" href="/Experimental-journal-deploy/assets/js/app.1f87a2af.js" as="script"><link rel="preload" href="/Experimental-journal-deploy/assets/js/2.1b349405.js" as="script"><link rel="preload" href="/Experimental-journal-deploy/assets/js/3.2bad108f.js" as="script"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/10.acf4644c.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/11.29af783f.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/12.64905565.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/13.feec988b.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/14.01c25098.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/15.6ee0e126.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/16.5fc24e5e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/17.d2deca79.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/18.4e9934a6.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/19.11f4f6a0.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/20.fdd5d708.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/21.42fe127d.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/22.ed381bc5.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/23.b9dff836.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/24.9554bf4e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/25.ae4c9030.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/26.4d80ce19.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/27.f81d4c61.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/28.76deaa42.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/29.1a21047e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/30.bdb5da0e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/31.f1e44357.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/32.d7568599.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/33.10a542e0.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/34.b6b65e19.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/35.39c507c4.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/36.3a15ed81.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/37.e77e50dc.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/38.24f056c7.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/39.3db2579e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/4.796b4f00.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/40.a04f14f9.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/41.99276188.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/42.198878ad.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/43.3d389bdf.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/44.fa00a393.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/45.2a58e897.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/46.39647e9a.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/47.509536f0.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/48.a7e3943c.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/49.5af5303a.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/5.7be9773c.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/50.c1b159e2.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/51.3db19103.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/52.23d4048e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/53.2ea76a1a.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/54.a05d750d.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/55.79515050.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/56.69432eaa.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/57.16a456e3.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/58.53e3eb33.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/59.c5841885.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/6.0676076f.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/60.21090696.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/61.49168cd4.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/62.f9c1a876.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/63.fce628cc.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/64.c1fed9f8.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/65.1fd23895.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/66.bf23ce03.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/67.05337aeb.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/68.24b62406.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/69.d9a47354.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/7.a456ae97.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/70.251612b8.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/71.2a42bbcd.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/72.dff336f6.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/73.9754d0c0.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/74.9a70f931.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/75.2d39661b.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/76.a4072598.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/77.3ed93e61.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/8.e1eaa9b3.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/9.f168d4af.js">
    <link rel="stylesheet" href="/Experimental-journal-deploy/assets/css/0.styles.1dc62a3a.css">
  </head>
  <body>
    <div id="app" data-server-rendered="true"><div class="theme-container"><header class="navbar"><div class="sidebar-button"><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="img" viewBox="0 0 448 512" class="icon"><path fill="currentColor" d="M436 124H12c-6.627 0-12-5.373-12-12V80c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12zm0 160H12c-6.627 0-12-5.373-12-12v-32c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12zm0 160H12c-6.627 0-12-5.373-12-12v-32c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12z"></path></svg></div> <a href="/Experimental-journal-deploy/" class="home-link router-link-active"><!----> <span class="site-name">Experimental-Journal-PhD</span></a> <div class="links"><div class="search-box"><input aria-label="Search" autocomplete="off" spellcheck="false" value=""> <!----></div> <!----></div></header> <div class="sidebar-mask"></div> <aside class="sidebar"><!---->  <ul class="sidebar-links"><li><a href="/Experimental-journal-deploy/" class="sidebar-link">Experimental progress 🔓 💎</a></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-05</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-06</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-07</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-08</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-09</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-10</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-01</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading open"><span>2020-02</span> <span class="arrow down"></span></p> <ul class="sidebar-links sidebar-group-items"><li><a href="/Experimental-journal-deploy/2020-02/2020-02-09-dnrp1-dbem2-for-satay.html" class="sidebar-link">Design of the genomic transformation to make dnrp1 and dbem2 strains usable for SATAY</a></li><li><a href="/Experimental-journal-deploy/2020-02/2020-02-13-Comparison-of Werner-FACs-settings-with-mine.html" class="sidebar-link">Comparison between Wener setting in FAcs with mine</a></li><li><a href="/Experimental-journal-deploy/2020-02/2020-02-21-hands-on-data-from-facs.html" class="sidebar-link">What are the output channels from flow cytometry actualling telling us about the fluorescence of every cell?</a></li><li><a href="/Experimental-journal-deploy/2020-02/2020-02-23-ylic133-5-transformations.html" class="active sidebar-link">Transformation of ylic133_5 with the PCR products from yll137 and yll140</a><ul class="sidebar-sub-headers"><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2020-02/2020-02-23-ylic133-5-transformations.html#date" class="sidebar-link">Date</a></li><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2020-02/2020-02-23-ylic133-5-transformations.html#objective" class="sidebar-link">Objective</a></li><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2020-02/2020-02-23-ylic133-5-transformations.html#method" class="sidebar-link">Method</a></li><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2020-02/2020-02-23-ylic133-5-transformations.html#results" class="sidebar-link">Results</a></li><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2020-02/2020-02-23-ylic133-5-transformations.html#conclusion" class="sidebar-link">Conclusion</a></li><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2020-02/2020-02-23-ylic133-5-transformations.html#next-steps" class="sidebar-link">Next Steps</a></li></ul></li><li><a href="/Experimental-journal-deploy/2020-02/2020-02-25-FACs-starting-with-30C-shaking.html" class="sidebar-link">FAcs-experiment-7</a></li></ul></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-03</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-04</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-05</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-06</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-07</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-08</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>General-Files</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>Protocols</span> <span class="arrow right"></span></p> <!----></section></li></ul> </aside> <main class="page"> <div class="theme-default-content content__default"><h1 id="title-transformation-of-ylic133-5-with-the-pcr-products-from-yll137-and-yll140"><a href="#title-transformation-of-ylic133-5-with-the-pcr-products-from-yll137-and-yll140" class="header-anchor">#</a> Title: Transformation of ylic133_5 with the PCR products from yll137 and yll140</h1> <h2 id="date"><a href="#date" class="header-anchor">#</a> Date</h2> <p>23022020-</p> <h2 id="objective"><a href="#objective" class="header-anchor">#</a> Objective</h2> <p>Transformation-1: To replace the nrp1 locus with the HYGRO cassette using the PCR product of yll137.</p> <p>Transformation-2: To replace bem2 locus with the leu2 marker using the PCR product of yll140.</p> <h2 id="method"><a href="#method" class="header-anchor">#</a> Method</h2> <ul><li><p>Following this <a href="/Experimental-journal-deploy/Protocols/Yeast-transformation.html">Yeast Transformation Protocol</a></p></li> <li><p>Day0: 16:00 , Incubation of ylic133_5A from glycerol stock from 01202019 in YPD+6xADE. 10mL per transformation with its negative control.</p></li> <li><p>Day1: Measure OD of overnight cultures</p> <table><thead><tr><th>Sample</th> <th>10X dilution OD</th> <th>Real OD</th></tr></thead> <tbody><tr><td>yll133_5 (1)</td> <td>0.1</td> <td>1</td></tr> <tr><td>yll133_5 (2)</td> <td>0.07</td> <td>0.7</td></tr></tbody></table></li> <li><p>At 10:00 am 2nd incubation with a 2x dilution down to OD~0.5</p></li> <li><p>Check at 14:00 the OD</p></li></ul> <table><thead><tr><th>Sample</th> <th>10X dilution OD</th> <th>Real OD</th></tr></thead> <tbody><tr><td>yll133_5 (1)</td> <td>0.28</td> <td>2.8</td></tr> <tr><td>yll133_5 (2)</td> <td>0.23</td> <td>2.3</td></tr></tbody></table> <ul><li>DNA amount for transformations
<ul><li>nrp1::HYGRO - 806 ng (73.3ng/ul*9ul)</li> <li>bem2::leu2 - 1.296 ug(144.4 ng/ul*9ul)</li></ul></li> <li>The plates to check the success of the transformations are:
<ul><li>YPD+HYGRO+6xADE</li> <li>-leu2+6xADE</li> <li>The only autoclave component was the agar (4g) with 100ml MiliQ, the rest was added by filtering (in 80ml MiliQ). To avoid degradation of aminoacids in the autoclave machine.</li></ul></li> <li><strong>Plating</strong>:
<ul><li>ylic133_5 + bem2::leu2   , low and high concentrated , 90x and 10x dilution, for a final volume of 200ul.</li></ul></li> <li><strong>Recovery step for ylic133_5+nrp1::HyGRO</strong> <ul><li>After removing the transformation mix , I added 1ml of YPD+6xADE to incubate in 30C overnight, and then plate that next morning on the selection plates.</li> <li>This choice for the recovery way is based on the experience when selecting using 5FOA plates for not having URA, <a href="/Experimental-journal-deploy/2019-07/2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html">here</a>.</li></ul></li></ul> <h2 id="results"><a href="#results" class="header-anchor">#</a> Results</h2> <p>25022020- Samples were spilled out from the eppi tube after the overnight incubation (I put too much YPD that cause the burst due to CO2 production)</p> <blockquote><p>Next time put parafilm, around it , to prevent spilling.</p></blockquote> <p><img src="/Experimental-journal-deploy/assets/img/25022020-samples-spilled-out.93263b78.png" alt="">{width=50%}</p> <p><strong>Note</strong>:</p> <blockquote><p>In general it is not advisable to wait more a whole overnight for recovery, because then we decrease the changes to have different biological replicates, since maybe one population will take over during the overnight incubation.</p></blockquote> <blockquote><p>Esentially the stadistics will go down , and if we have suppresor they will be &quot;hidden&quot; when plating on the selection plates.</p></blockquote> <ul><li><p>I got more than 50 transformants in the -leu2 plates and a big splash of cells combiend with colonies in one of the hygro plate.</p> <ul><li>I inoculated 200ul from the eppi tube that spilled out, onto the two plates, without any dilution.</li> <li>In one of the plates there was a big loan because they were many colonies on it.</li> <li>I did not see any bacterial contamination on the plates. 😃</li></ul></li> <li><p>Colony PCR to check the transformation</p> <ul><li>I check 8 colonies from the hygro plate and -leu2 plate and retreak them onto a new plate, to save the colony.
<ul><li>First colony PCR I did using SPY1/2 Els primers and 1ul of template.</li></ul></li></ul></li></ul> <p><img src="/Experimental-journal-deploy/assets/img/27022020-colony-pcr-nrp1&amp;bem2.77d38c02.png" alt="1st Colony PCR gel with 1ul template">{#fig:colony_pcr_1}</p> <p><img src="/Experimental-journal-deploy/assets/img/28022020-colony-pcr-nrp1&amp;bem2-small-colonies.53b53389.png" alt="2nd Colony PCR with 5ul template and more bem2:leu2 colonies. The second half of the gel after the 3a with 33/38 primers corresponds with the 9-16 small colonies from -leu2 plates.">{#fig:colony_pcr_2}</p> <ul><li><p>For the nrp1::hygro clones it has not worked with SPY1/2 primers.</p></li> <li><p>Colony PCR for nrp1::hygro clones with 27/32 primers from my stock and 83/84 from Els.</p> <ul><li>I take single colonies from the restreak onto Hygro.
<img src="/Experimental-journal-deploy/assets/img/02032020-colony-pcr-nrp1-hygro-different-primers.49af07cf.png" alt=""></li> <li>It is not clear whether the nrp1::hygro have the right length. Neither the negative control with primers 83/84.</li> <li>[x] Do another PCR with sequencing primers to check again. 03032020
<ul><li>I did the PCR with the sequencing primers and redo the gel for the previous PCR with primers 27/32 and 83/84 but running the gel 40 mins at 110V. (Following Els and Liedewij recommendations from lab meeting). Below are the results:</li></ul></li></ul> <p><img src="/Experimental-journal-deploy/assets/img/03032020-Re-PCR-02032020&amp;sequencing-primers.7240d36b.png" alt="Large gel showing that increasing the time of the gel allow to resolve closer bands. Colony 1 and 8 from nrp1::hygro clones seems have both(?), and the rest seems OK according the band lenght. This observation is also consistent with primers 83 and 84. No bands are shown for the sequencing primers for the nrp1::hygro clones.For bem2 it seems also OK, all the bands. Still yll3a does not show the right band in 7000bp which could be due to the PCR cycle.">{#fig:seq-PCR}</p> <ul><li>Results:
<ul><li>Colony 1 and 8 from nrp1::hygro clones seems have both(?), and the rest seems OK according the band lenght. This observation is also consistent with primers 83 and 84. No bands are shown for the sequencing primers for the nrp1::hygro clones.</li> <li>For bem2 it seems also OK, all the bands. Still yll3a does not show the right band in 7000bp which could be due to the PCR cycle.</li></ul></li></ul></li> <li><p>I took single colonies (8,12,13,16) from the -leu2 plate restreak and grow them in YPD+6xADE  for glycerol stock and restreak them in normal YPD plate to check for the red phenotype.</p></li></ul> <p><img src="/Experimental-journal-deploy/assets/img/bem2-colonies-8,12,13,16-growth-48h.6f3938df.png" alt="Indeed they are red :) "></p> <ul><li>[x] Do colony PCR with SPY1-bem2 and SPY2-bem2 with all small and big colonies that have shown a band with 33/38 primers.</li> <li>[x] Store also 5 small and  5 big colonies to later test the population phenotype, and decided which is bem2d. Because it seems I have two populations, that show the right band on gel.</li></ul> <p><img src="/Experimental-journal-deploy/assets/img/04032020-hygro-nrp1-checking-frm-inside-SPY-bem2.9a8a6a36.png" alt="Gel showing the nrp1 transformants from 2-6 results from a PCR testing the hygro marker and the lack of nrp1 in the same region."></p> <p><img src="/Experimental-journal-deploy/assets/img/05032020-hgro-nrp1-inside.e676e8b0.png" alt="Gel showing that the nrp1 clones 2-6 are GOOD!!  so they passed this test of checking whther they have hygro marker or not."></p> <h2 id="conclusion"><a href="#conclusion" class="header-anchor">#</a> Conclusion</h2> <ul><li><p>It seems I could take 5 clones per transformation that seems they are correct 😃</p></li> <li><p>[x] I sent already (05032020) bem2::leu2 clones to sequence with primers SPY1 and SPY2 (good results)</p> <p><img src="/Experimental-journal-deploy/assets/img/24032020-S288C_chrV_bem2-leu2-seq-Map.9cf0ff5b.png" alt=""></p></li> <li><p>[x] I will send today (06033030) nrp1::hygro clones to sequencing.</p> <p><img src="/Experimental-journal-deploy/assets/img/24032020-S288C_chrIV_nrp1-Hygro-seq-Map.17a50857.png" alt=""></p></li> <li><p>[x] I am also incubating those clones in liquid selection media to do a population phenotype test in the Biotek.
Results 😃</p> <p><img src="/Experimental-journal-deploy/assets/img/dnrp1-biolg-replicates.965f400c.png" alt=""></p> <p><img src="/Experimental-journal-deploy/assets/img/dbem2-biolg-replicates.7adbccc8.png" alt=""></p> <ul><li><p>The code I wrote to compute the doubling times is in this <a href="C:%5CUsers%5Clinigodelacruz%5CDocuments%5CPhD_2018%5CDocumentation%5CSATAY%5Cnrp1-bem2-single-deletions-for-satay-02-2020%5CBiotek-09032020%5Cbiotek-data-analysis.ipynb">jupyter notebook</a></p> <p><img src="/Experimental-journal-deploy/assets/img/dnrp1-dbem2-growth-rates.fc0cc576.png" alt="">{width=60%}</p></li> <li><p>Indeed there is less variation among nrp1d clones than bem2d clones, which could be due to the overnight recovery.</p></li></ul></li> <li><p>I will also stored them today as glycerol stocks in the selection media.</p> <ul><li>I stored only one tube per biolog replicate.</li></ul></li></ul> <h2 id="next-steps"><a href="#next-steps" class="header-anchor">#</a> Next Steps</h2> <ul><li>[ ] Repeat the biotek measurements using the wt strain in order to compare these mutants relative to the WT.</li> <li>[ ] Make a PCR from inside the leu2 marker and bem2 to check the correct integration of bem2::leu2 transformants. (colony PCR) <strong>still is not clear the differences I see in the selected biological replicates for bem2::leu2 mutants</strong></li></ul></div> <footer class="page-edit"><!----> <!----></footer> <div class="page-nav"><p class="inner"><span class="prev">
      ←
      <a href="/Experimental-journal-deploy/2020-02/2020-02-21-hands-on-data-from-facs.html" class="prev">What are the output channels from flow cytometry actualling telling us about the fluorescence of every cell?</a></span> <span class="next"><a href="/Experimental-journal-deploy/2020-02/2020-02-25-FACs-starting-with-30C-shaking.html">FAcs-experiment-7</a>
      →
    </span></p></div> </main></div><div class="global-ui"></div></div>
    <script src="/Experimental-journal-deploy/assets/js/app.1f87a2af.js" defer></script><script src="/Experimental-journal-deploy/assets/js/2.1b349405.js" defer></script><script src="/Experimental-journal-deploy/assets/js/3.2bad108f.js" defer></script>
  </body>
</html>
