<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>16-07-2019-Script in python to find the most connected genes to a query gene. 🌌 | Experimental-Journal-PhD</title>
    <meta name="description" content="">
    
    
    <link rel="preload" href="/Experimental-journal-deploy/assets/css/0.styles.1dc62a3a.css" as="style"><link rel="preload" href="/Experimental-journal-deploy/assets/js/app.1f87a2af.js" as="script"><link rel="preload" href="/Experimental-journal-deploy/assets/js/2.1b349405.js" as="script"><link rel="preload" href="/Experimental-journal-deploy/assets/js/29.1a21047e.js" as="script"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/10.acf4644c.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/11.29af783f.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/12.64905565.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/13.feec988b.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/14.01c25098.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/15.6ee0e126.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/16.5fc24e5e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/17.d2deca79.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/18.4e9934a6.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/19.11f4f6a0.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/20.fdd5d708.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/21.42fe127d.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/22.ed381bc5.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/23.b9dff836.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/24.9554bf4e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/25.ae4c9030.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/26.4d80ce19.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/27.f81d4c61.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/28.76deaa42.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/3.2bad108f.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/30.bdb5da0e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/31.f1e44357.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/32.d7568599.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/33.10a542e0.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/34.b6b65e19.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/35.39c507c4.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/36.3a15ed81.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/37.e77e50dc.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/38.24f056c7.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/39.3db2579e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/4.796b4f00.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/40.a04f14f9.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/41.99276188.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/42.198878ad.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/43.3d389bdf.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/44.fa00a393.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/45.2a58e897.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/46.39647e9a.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/47.509536f0.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/48.a7e3943c.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/49.5af5303a.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/5.7be9773c.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/50.c1b159e2.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/51.3db19103.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/52.23d4048e.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/53.2ea76a1a.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/54.a05d750d.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/55.79515050.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/56.69432eaa.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/57.16a456e3.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/58.53e3eb33.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/59.c5841885.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/6.0676076f.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/60.21090696.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/61.49168cd4.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/62.f9c1a876.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/63.fce628cc.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/64.c1fed9f8.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/65.1fd23895.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/66.bf23ce03.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/67.05337aeb.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/68.24b62406.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/69.d9a47354.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/7.a456ae97.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/70.251612b8.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/71.2a42bbcd.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/72.dff336f6.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/73.9754d0c0.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/74.9a70f931.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/75.2d39661b.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/76.a4072598.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/77.3ed93e61.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/8.e1eaa9b3.js"><link rel="prefetch" href="/Experimental-journal-deploy/assets/js/9.f168d4af.js">
    <link rel="stylesheet" href="/Experimental-journal-deploy/assets/css/0.styles.1dc62a3a.css">
  </head>
  <body>
    <div id="app" data-server-rendered="true"><div class="theme-container"><header class="navbar"><div class="sidebar-button"><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="img" viewBox="0 0 448 512" class="icon"><path fill="currentColor" d="M436 124H12c-6.627 0-12-5.373-12-12V80c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12zm0 160H12c-6.627 0-12-5.373-12-12v-32c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12zm0 160H12c-6.627 0-12-5.373-12-12v-32c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12z"></path></svg></div> <a href="/Experimental-journal-deploy/" class="home-link router-link-active"><!----> <span class="site-name">Experimental-Journal-PhD</span></a> <div class="links"><div class="search-box"><input aria-label="Search" autocomplete="off" spellcheck="false" value=""> <!----></div> <!----></div></header> <div class="sidebar-mask"></div> <aside class="sidebar"><!---->  <ul class="sidebar-links"><li><a href="/Experimental-journal-deploy/" class="sidebar-link">Experimental progress 🔓 💎</a></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-05</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-06</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading open"><span>2019-07</span> <span class="arrow down"></span></p> <ul class="sidebar-links sidebar-group-items"><li><a href="/Experimental-journal-deploy/2019-07/2019-07-04_Exp_pBK549_digestion_plasmid.html" class="sidebar-link">Title : 03072019-Digestion pattern of pBK549 (Succesful) :ok:</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-08-Data-ylic132_biolog-replicates-sequencing.html" class="sidebar-link">Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 :ok: :whitecheckmark:</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-10_Byk832-transformation-pBk549.html" class="sidebar-link">Title : 10072019-Byk832 transformation using pBK549 plasmid (:ok:)</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html" class="sidebar-link">Title : 11072019-ylic132-1 and ylic132-2  transformation with the OEP2 :whitecheckmark:</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-16-Byk832-checking-clones-ade-ura.html" class="sidebar-link">Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549 :ok:</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-16_common-interactor-genes-workflow.html" class="active sidebar-link">16-07-2019-Script in python to find the most connected genes to a query gene. 🌌</a><ul class="sidebar-sub-headers"><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2019-07/2019-07-16_common-interactor-genes-workflow.html#_16-07-2019-script-in-python-to-find-the-most-connected-genes-to-a-query-gene" class="sidebar-link">16-07-2019-Script in python to find the most connected genes to a query gene. 🌌</a></li><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2019-07/2019-07-16_common-interactor-genes-workflow.html#python-code" class="sidebar-link">Python code</a></li><li class="sidebar-sub-header"><a href="/Experimental-journal-deploy/2019-07/2019-07-16_common-interactor-genes-workflow.html#results-figure" class="sidebar-link">Results Figure</a></li></ul></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-17_Exp-Genomic-prep-ylic133-clones.html" class="sidebar-link">Title : 17072019-DNA extraction of ylic133: ura3- ade2- clones anad PCR 🆗 ✅</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-19-glycerol-stocks-ylic133-bio-replicates.html" class="sidebar-link">Title : 23072019-Glycerols stocks of ylic133 biological replicates :snowflake:</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-19_Sequencing-ylic133-biol-replicates.html" class="sidebar-link">Title: 22072019-Sequencing of ylic133 biological replicates :ok: :whitecheckmark:</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-23-SATAY-Byk832-1st-attempt.html" class="sidebar-link">Title: 29072019-SATAY 1st attempt with Byk832-T :pray: :confused: :x:</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-24-Strategy-to-SATAY-target-strains.html" class="sidebar-link">Title : 24072019-Strategies to generate the strains I want to study using SATAY :pagewithcurl:</a></li><li><a href="/Experimental-journal-deploy/2019-07/2019-07-31-Extra-checking-steps-SATAY-Byk832T.html" class="sidebar-link">Title: 31072019 Double check Byk832T biolog.  replicates suitability for SATAY.</a></li></ul></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-08</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-09</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2019-10</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-01</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-02</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-03</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-04</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-05</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-06</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-07</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>2020-08</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>General-Files</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>Protocols</span> <span class="arrow right"></span></p> <!----></section></li></ul> </aside> <main class="page"> <div class="theme-default-content content__default"><h2 id="_16-07-2019-script-in-python-to-find-the-most-connected-genes-to-a-query-gene"><a href="#_16-07-2019-script-in-python-to-find-the-most-connected-genes-to-a-query-gene" class="header-anchor">#</a> 16-07-2019-Script in python to find the most connected genes to a query gene. 🌌</h2> <h3 id="goal"><a href="#goal" class="header-anchor">#</a> Goal</h3> <p>From the physical and genetic interactors of a specific gene, I am interested in finding which of the interactors of the interactors are also interactors of the initial query gene.</p> <p>This calculation can maybe triggers hypothesis about how  is the correlation in the amount of connectivity of one gene to another gene and the type of interactions they are more prone to share.</p> <p>Perhaps, it is like this:
<img src="/Experimental-journal-deploy/assets/img/Model_cartoon-05.e2871f42.png" alt=""></p> <p>The following picture depicts what I am looking for in the <a href="https://www.yeastgenome.org/" target="_blank" rel="noopener noreferrer">SGD database<svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" x="0px" y="0px" viewBox="0 0 100 100" width="15" height="15" class="icon outbound"><path fill="currentColor" d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"></path> <polygon fill="currentColor" points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"></polygon></svg></a> for every gene of interest:
<img src="/Experimental-journal-deploy/assets/img/cartoon-for-common-genes-workflow07.2fa48284.png" alt=""></p> <p>The following python code depicts what I have done, that can be reused and improved  for other purposes.</p> <h2 id="python-code"><a href="#python-code" class="header-anchor">#</a> Python code</h2> <div class="language-python extra-class"><pre class="language-python"><code><span class="token keyword">import</span> pandas <span class="token keyword">as</span> pd
<span class="token keyword">import</span> numpy <span class="token keyword">as</span> np
<span class="token keyword">from</span> collections <span class="token keyword">import</span> defaultdict
<span class="token keyword">import</span> seaborn <span class="token keyword">as</span> sns
<span class="token keyword">import</span> matplotlib<span class="token punctuation">.</span>pyplot <span class="token keyword">as</span> plt
<span class="token operator">%</span>matplotlib inline


<span class="token comment">## good website to study dataframes</span>
<span class="token comment">#https://www.shanelynn.ie/using-pandas-dataframe-creating-editing-viewing-data-in-python/</span>
</code></pre></div><div class="language-python extra-class"><pre class="language-python"><code>data<span class="token operator">=</span>pd<span class="token punctuation">.</span>read_excel<span class="token punctuation">(</span><span class="token string">r'C:\Users\linigodelacruz\Documents\PhD_2018\Documentation\Calculations\data_sgd\Interaction_data_sgd_downloads.xlsx'</span><span class="token punctuation">,</span>header<span class="token operator">=</span><span class="token number">17</span><span class="token punctuation">,</span>encoding<span class="token operator">=</span><span class="token string">&quot;utf-8-sig&quot;</span><span class="token punctuation">)</span>
col_label<span class="token operator">=</span>data<span class="token punctuation">.</span>columns<span class="token punctuation">.</span>values
data_go<span class="token operator">=</span>pd<span class="token punctuation">.</span>read_excel<span class="token punctuation">(</span><span class="token string">r'C:\Users\linigodelacruz\Documents\PhD_2018\Documentation\Calculations\data_sgd\slim_goterms_data_sgd_downloads.xlsx'</span><span class="token punctuation">,</span>header<span class="token operator">=</span><span class="token number">14</span><span class="token punctuation">,</span>encoding<span class="token operator">=</span><span class="token string">&quot;utf-8-sig&quot;</span><span class="token punctuation">)</span>
col_label_go<span class="token operator">=</span>data_go<span class="token punctuation">.</span>columns<span class="token punctuation">.</span>values
data_go<span class="token punctuation">.</span>set_index<span class="token punctuation">(</span>col_label_go<span class="token punctuation">[</span><span class="token number">1</span><span class="token punctuation">]</span><span class="token punctuation">,</span>inplace<span class="token operator">=</span><span class="token boolean">True</span><span class="token punctuation">)</span>
</code></pre></div><div class="language-python extra-class"><pre class="language-python"><code>


d2 <span class="token operator">=</span> defaultdict<span class="token punctuation">(</span><span class="token builtin">dict</span><span class="token punctuation">)</span>
query<span class="token operator">=</span><span class="token punctuation">[</span><span class="token string">'BEM1'</span><span class="token punctuation">]</span> <span class="token comment"># here you can put the gene of interest</span>
<span class="token comment"># giant for loop</span>
names1 <span class="token operator">=</span> query
i<span class="token operator">=</span><span class="token operator">-</span><span class="token number">1</span>
<span class="token keyword">for</span> query1 <span class="token keyword">in</span> names1<span class="token punctuation">:</span>
    <span class="token comment">#filtering the table just for the value of the query</span>
    q1 <span class="token operator">=</span> data<span class="token punctuation">[</span>data<span class="token punctuation">[</span><span class="token string">'Standard_Gene_Name_(Bait)'</span><span class="token punctuation">]</span><span class="token operator">==</span>query1<span class="token punctuation">]</span>
    q1_interact<span class="token operator">=</span>q1<span class="token punctuation">[</span>col_label<span class="token punctuation">[</span><span class="token number">3</span><span class="token punctuation">]</span><span class="token punctuation">]</span><span class="token punctuation">.</span>unique<span class="token punctuation">(</span><span class="token punctuation">)</span>
   <span class="token comment"># a for loop for all the interactors of query</span>
    <span class="token keyword">for</span> query2 <span class="token keyword">in</span> q1_interact<span class="token punctuation">:</span>


        q2<span class="token operator">=</span>data<span class="token punctuation">[</span>data<span class="token punctuation">[</span><span class="token string">'Standard_Gene_Name_(Bait)'</span><span class="token punctuation">]</span><span class="token operator">==</span>query2<span class="token punctuation">]</span> <span class="token comment">#these are get_query(q1[i])</span>
        q2_interact<span class="token operator">=</span>q2<span class="token punctuation">[</span>col_label<span class="token punctuation">[</span><span class="token number">3</span><span class="token punctuation">]</span><span class="token punctuation">]</span><span class="token punctuation">.</span>unique<span class="token punctuation">(</span><span class="token punctuation">)</span>
        d <span class="token operator">=</span> defaultdict<span class="token punctuation">(</span><span class="token builtin">int</span><span class="token punctuation">)</span>
        common <span class="token operator">=</span> <span class="token punctuation">[</span><span class="token punctuation">]</span>

        <span class="token keyword">for</span> name1  <span class="token keyword">in</span> q2_interact<span class="token punctuation">:</span>
            <span class="token keyword">if</span> name1 <span class="token keyword">in</span> q1_interact<span class="token punctuation">:</span> <span class="token comment"># if a gene interactor of the query1 is in interactors of query 2</span>
                common<span class="token punctuation">.</span>append<span class="token punctuation">(</span>name1<span class="token punctuation">)</span>
                d<span class="token punctuation">[</span>name1<span class="token punctuation">]</span> <span class="token operator">+=</span> <span class="token number">1</span>


        d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span> query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;common&quot;</span><span class="token punctuation">]</span> <span class="token operator">=</span> common
        d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span>query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;names of genes&quot;</span><span class="token punctuation">]</span><span class="token operator">=</span>query2
        d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span> query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;n_common&quot;</span><span class="token punctuation">]</span> <span class="token operator">=</span> <span class="token builtin">len</span><span class="token punctuation">(</span>common<span class="token punctuation">)</span>
        d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span> query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;len_i_1&quot;</span><span class="token punctuation">]</span> <span class="token operator">=</span> <span class="token builtin">len</span><span class="token punctuation">(</span>q1<span class="token punctuation">)</span>
        d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span> query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;len_i_2&quot;</span><span class="token punctuation">]</span> <span class="token operator">=</span> <span class="token builtin">len</span><span class="token punctuation">(</span>q2<span class="token punctuation">)</span>
        <span class="token keyword">if</span> <span class="token builtin">len</span><span class="token punctuation">(</span>q1<span class="token punctuation">)</span><span class="token operator">==</span><span class="token number">0</span><span class="token punctuation">:</span>
            d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span> query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;% of query subset&quot;</span><span class="token punctuation">]</span> <span class="token operator">=</span> <span class="token number">0</span>
        <span class="token keyword">else</span><span class="token punctuation">:</span>
            d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span> query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;% of query subset&quot;</span><span class="token punctuation">]</span> <span class="token operator">=</span> <span class="token builtin">len</span><span class="token punctuation">(</span>d<span class="token punctuation">)</span><span class="token operator">/</span><span class="token builtin">len</span><span class="token punctuation">(</span>q1_interact<span class="token punctuation">)</span> <span class="token operator">*</span><span class="token number">100</span>

        <span class="token keyword">if</span> <span class="token builtin">len</span><span class="token punctuation">(</span>q2<span class="token punctuation">)</span><span class="token operator">==</span><span class="token number">0</span><span class="token punctuation">:</span>
            d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span> query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;% of query 2 subset  &quot;</span><span class="token punctuation">]</span> <span class="token operator">=</span> <span class="token number">0</span>
        <span class="token keyword">else</span><span class="token punctuation">:</span>
            d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span> query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;% of query 2 subset  &quot;</span><span class="token punctuation">]</span> <span class="token operator">=</span> <span class="token builtin">len</span><span class="token punctuation">(</span>d<span class="token punctuation">)</span><span class="token operator">/</span><span class="token builtin">len</span><span class="token punctuation">(</span>q2_interact<span class="token punctuation">)</span> <span class="token operator">*</span><span class="token number">100</span>

            q1_filt<span class="token operator">=</span>q1<span class="token punctuation">[</span>q1<span class="token punctuation">[</span>col_label<span class="token punctuation">[</span><span class="token number">3</span><span class="token punctuation">]</span><span class="token punctuation">]</span><span class="token operator">==</span>query2<span class="token punctuation">]</span>
            d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span>query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">&quot;interact_annotation&quot;</span><span class="token punctuation">]</span><span class="token operator">=</span>q1_filt<span class="token punctuation">[</span>col_label<span class="token punctuation">[</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">]</span>
            d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span>query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">'GO_slim_query'</span><span class="token punctuation">]</span><span class="token operator">=</span> data_go<span class="token punctuation">.</span>loc<span class="token punctuation">[</span>query1<span class="token punctuation">]</span><span class="token punctuation">[</span>col_label_go<span class="token punctuation">[</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">]</span>
            <span class="token comment">#d2[query1,query2]['GO_slim_interactors']= data_go[data_go[col_label_go[1]]==query2][col_label_go[4]]</span>

            d2<span class="token punctuation">[</span>query1<span class="token punctuation">,</span>query2<span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token string">'GO_slim_interactors'</span><span class="token punctuation">]</span><span class="token operator">=</span> data_go<span class="token punctuation">.</span>loc<span class="token punctuation">[</span>query2<span class="token punctuation">]</span><span class="token punctuation">[</span>col_label_go<span class="token punctuation">[</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">]</span>


</code></pre></div><div class="language-python extra-class"><pre class="language-python"><code>df<span class="token operator">=</span>pd<span class="token punctuation">.</span>DataFrame<span class="token punctuation">(</span>d2<span class="token punctuation">)</span><span class="token punctuation">.</span>T
df
</code></pre></div><div><style scoped="scoped">
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }
&lt;pre&gt;&lt;code&gt;.dataframe tbody tr th {
    vertical-align: top;
}

.dataframe thead th {
    text-align: right;
}
&lt;/code&gt;&lt;/pre&gt;
</style> <table border="1" class="dataframe"><thead><tr style="text-align:right;"><th></th> <th></th> <th>% of query 2 subset</th> <th>% of query subset</th> <th>GO_slim_interactors</th> <th>GO_slim_query</th> <th>common</th> <th>interact_annotation</th> <th>len_i_1</th> <th>len_i_2</th> <th>n_common</th> <th>names of genes</th></tr></thead> <tbody><tr><th rowspan="61" valign="top">BEM1</th> <th>ATS1</th> <td>15.2542</td> <td>2.80374</td> <td>Gene (optional)
ATS1                    cytopl...</td> <td>Gene (optional)
BEM1                    cellul...</td> <td>[RPS8A, BEM1, STE50, RVS161, SAC3, ROM2, TPM1,...</td> <td>4872    Negative Genetic
Name: Experiment_Type...</td> <td>415</td> <td>66</td> <td>9</td> <td>ATS1</td></tr> <tr><th>PMT2</th> <td>23.6364</td> <td>8.09969</td> <td>Gene (optional)
PMT2                          ...</td> <td>Gene (optional)
BEM1                    cellul...</td> <td>[PEP1, ECM33, PMT1, RGP1, ERD1, GET2, GLO3, PM...</td> <td>5487    Negative Genetic
Name: Experiment_Type...</td> <td>415</td> <td>143</td> <td>26</td> <td>PMT2</td></tr> <tr><th>LTE1</th> <td>13.8264</td> <td>13.3956</td> <td>Gene (optional)
LTE1                          ...</td> <td>Gene (optional)
BEM1                    cellul...</td> <td>[CDC24, SWD1, FUS3, UBP14, BEM1, MBP1, BRE1, S...</td> <td>5830    Negative Genetic
Name: Experiment_Type...</td> <td>415</td> <td>367</td> <td>43</td> <td>LTE1</td></tr> <tr><th>SNC1</th> <td>44</td> <td>3.42679</td> <td>Gene (optional)
SNC1             Golgi apparat...</td> <td>Gene (optional)
BEM1                    cellul...</td> <td>[SEC17, SEC18, CYK3, ARF1, SWF1, VPS1, ELO3, S...</td> <td>7080    Affinity Capture-Western
Name: Experim...</td> <td>415</td> <td>50</td> <td>11</td> <td>SNC1</td></tr> <tr><th>CLN3</th> <td>35.7143</td> <td>7.78816</td> <td>Gene (optional)
CLN3                          ...</td> <td>Gene (optional)
BEM1                    cellul...</td> <td>[STE50, MBP1, STE5, RPA14, MNN10, SSD1, IPK1, ...</td> <td>7971    Positive Genetic
Name: Experiment_Type...</td> <td>415</td> <td>106</td> <td>25</td> <td>CLN3</td></tr> <tr><th>CDC24</th> <td>27.4809</td> <td>11.215</td> <td>Gene (optional)
CDC24                         ...</td> <td>Gene (optional)
BEM1                    cellul...</td> <td>[CDC24, BOI1, BEM1, RDI1, STE5, GIC2, RGA2, SA...</td> <td>8231         Synthetic Lethality
8232    Affin...</td> <td>415</td> <td>200</td> <td>36</td> <td>CDC24</td></tr> <tr><th>SWD1</th> <td>18.8811</td> <td>8.41121</td> <td>Gene (optional)
SWD1                          ...</td> <td>Gene (optional)
BEM1                    cellul...</td> <td>[RXT2, STE50, BRE1, RPA14, SAC3, MSN5, SEM1, V...</td> <td>10815    Positive Genetic
Name: Experiment_Typ...</td> <td>415</td> <td>200</td> <td>27</td> <td>SWD1</td></tr> <tr><th>BUD14</th> <td>26.7442</td> <td>7.16511</td> <td>Gene (optional)
      BUD14                         ...</td> <td>Gene (optional)
      BEM1                    cellul...</td> <td>[RXT2, RRP7, STE50, TPS2, MTH1, IPK1, SWI4, BE...</td> <td>11634    Positive Genetic
      Name: Experiment_Typ...</td> <td>415</td> <td>99</td> <td>23</td> <td>BUD14</td></tr> <tr><th>PDR3</th> <td>17.5</td> <td>2.18069</td> <td>Gene (optional)
      PDR3                          ...</td> <td>Gene (optional)
      BEM1                    cellul...</td> <td>[MNN10, SEM1, SWI4, BEM2, MNN11, OPI3, ELM1]</td> <td>13524    Two-hybrid
      Name: Experiment_Type_(Req...</td> <td>415</td> <td>41</td> <td>7</td> <td>PDR3</td></tr> <tr><th>FUS3</th> <td>23.4177</td> <td>11.5265</td> <td>Gene (optional)
      FUS3                          ...</td> <td>Gene (optional)
      BEM1                    cellul...</td> <td>[CLN3, BUD14, FUS3, BOI1, RXT2, BEM1, STE50, S...</td> <td>15220    Affinity Capture-Western
      Name: Experi...</td> <td>415</td> <td>249</td> <td>37</td> <td>FUS3</td></tr> <tr><th>PEP1</th> <td>15.7895</td> <td>1.86916</td> <td>Gene (optional)
      PEP1        Golgi apparatus
      PE...</td> <td>Gene (optional)
      BEM1                    cellul...</td> <td>[PEP1, PMR1, RIM101, RIC1, RIM21, VPS5]</td> <td>15656    Negative Genetic
      Name: Experiment_Typ...</td> <td>415</td> <td>45</td> <td>6</td> <td>PEP1</td></tr> <tr><th>SEC17</th> <td>23.7624</td> <td>7.47664</td> <td>Gene (optional)
      SEC17                    cytop...</td> <td>Gene (optional)
      BEM1                    cellul...</td> <td>[SEC18, SWF1, MSN5, GET2, VAM7, IST3, CNB1, YK...</td> <td>19834    Co-fractionation
      Name: Experiment_Typ...</td> <td>415</td> <td>114</td> <td>24</td> <td>SEC17</td></tr> <tr><th>SKT5</th> <td>33.6364</td> <td>11.5265</td> <td>Gene (optional)
      SKT5                          ...</td> <td>Gene (optional)
      BEM1                    cellul...</td> <td>[RPS8A, CHS3, ECM33, BEM1, STE50, RVS161, CYK3...</td> <td>20873    Negative Genetic
      Name: Experiment_Typ...</td> <td>415</td> <td>169</td> <td>37</td> <td>SKT5</td></tr> <tr><th>RPS8A</th> <td>0</td> <td>0</td> <td>Gene (optional)
      RPS8A                         ...</td> <td>Gene (optional)
      BEM1                    cellul...</td> <td>[]</td> <td>21501    Negative Genetic
      Name: Experiment_Typ...</td> <td>415</td> <td>5</td> <td>0</td> <td>RPS8A</td></tr> <tr><th>BOI1</th> <td>27.2727</td> <td>4.6729</td> <td>Gene (optional)
      BOI1                cellular b...</td> <td>Gene (optional)
      BEM1                    cellul...</td> <td>[EXO84, BEM1, RRP7, MTH1, SEC3, BOI2, BEM2, CD...</td> <td>22819       Negative Genetic
      22820    Affinity...</td> <td>415</td> <td>78</td> <td>15</td> <td>BOI1</td></tr> <tr><th>NTH2</th> <td>16.6667</td> <td>0.934579</td> <td>Gene (optional)
            NTH2                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BEM1, STE50, TPS2]</td> <td>24979    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>20</td> <td>3</td> <td>NTH2</td></tr> <tr><th>MNN2</th> <td>36.6197</td> <td>8.09969</td> <td>Gene (optional)
            MNN2                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[PMT2, MNN2, VAM6, PMT1, ARF1, ERD1, GET2, SEC...</td> <td>27420    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>82</td> <td>26</td> <td>MNN2</td></tr> <tr><th>CHS3</th> <td>30.9735</td> <td>10.9034</td> <td>Gene (optional)
            CHS3                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[CDC24, SKT5, RPS8A, MNN2, CHS3, VAM6, ARF1, M...</td> <td>27890    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>182</td> <td>35</td> <td>CHS3</td></tr> <tr><th>RPS11B</th> <td>7.69231</td> <td>0.623053</td> <td>Gene (optional)
            RPS11B                        ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BEM1, ARF1]</td> <td>30303    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>26</td> <td>2</td> <td>RPS11B</td></tr> <tr><th>UBP14</th> <td>22.9885</td> <td>6.23053</td> <td>Gene (optional)
            UBP14                         ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[CYC8, BEM1, STE50, RVS161, MNN10, LSM6, RVS16...</td> <td>30960    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>92</td> <td>20</td> <td>UBP14</td></tr> <tr><th>ECM33</th> <td>30.5389</td> <td>15.8879</td> <td>Gene (optional)
            ECM33                         ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[PMT2, PEP1, SKT5, BEM1, STE50, MBP1, VAM6, PM...</td> <td>32618    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>184</td> <td>51</td> <td>ECM33</td></tr> <tr><th>SEC18</th> <td>16.25</td> <td>4.04984</td> <td>Gene (optional)
            SEC18             Golgi appara...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[SEC18, SSD1, LAS21, PBS2, RPS21B, CNB1, YKT6,...</td> <td>33169    Co-fractionation
            Name: Experiment_Typ...</td> <td>415</td> <td>85</td> <td>13</td> <td>SEC18</td></tr> <tr><th>RXT2</th> <td>24.1935</td> <td>9.34579</td> <td>Gene (optional)
            RXT2                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[LTE1, FUS3, STE50, RVS161, SAC3, SSD1, SEM1, ...</td> <td>35601    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>174</td> <td>30</td> <td>RXT2</td></tr> <tr><th>EXO84</th> <td>37.931</td> <td>3.42679</td> <td>Gene (optional)
            EXO84                   cell c...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[EXO84, SEC5, SEC3, SEC4, SEC15, SEC6, EXO70, ...</td> <td>36506    Co-purification
            Name: Experiment_Type...</td> <td>415</td> <td>56</td> <td>11</td> <td>EXO84</td></tr> <tr><th>CYC8</th> <td>11.2903</td> <td>2.18069</td> <td>Gene (optional)
            CYC8                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[CYC8, SAC3, SKN7, SWI6, ROM2, YDJ1, SIN3]</td> <td>38572    Two-hybrid
            Name: Experiment_Type_(Req...</td> <td>415</td> <td>78</td> <td>7</td> <td>CYC8</td></tr> <tr><th>BEM1</th> <td>100</td> <td>100</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[ATS1, PMT2, LTE1, SNC1, CLN3, CDC24, SWD1, BU...</td> <td>48372    PCA
            Name: Experiment_Type_(Required) ...</td> <td>415</td> <td>415</td> <td>321</td> <td>BEM1</td></tr> <tr><th>LDB16</th> <td>0</td> <td>0</td> <td>NaN</td> <td>NaN</td> <td>[]</td> <td>NaN</td> <td>415</td> <td>0</td> <td>0</td> <td>LDB16</td></tr> <tr><th>RRP7</th> <td>4.16667</td> <td>0.623053</td> <td>Gene (optional)
            RRP7                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[RPS27B, RIC1]</td> <td>48377    Two-hybrid
            Name: Experiment_Type_(Req...</td> <td>415</td> <td>51</td> <td>2</td> <td>RRP7</td></tr> <tr><th>STE50</th> <td>12.3711</td> <td>3.73832</td> <td>Gene (optional)
            STE50                     cyto...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[STE50, VAM6, STE5, PSP1, PTP3, VAM7, KEL2, ST...</td> <td>48378    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>127</td> <td>12</td> <td>STE50</td></tr> <tr><th>RVS161</th> <td>35.1724</td> <td>15.8879</td> <td>Gene (optional)
            RVS161                     cel...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[CLN3, BUD14, SKT5, MNN2, CHS3, RXT2, BEM1, CY...</td> <td>48381    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>262</td> <td>51</td> <td>RVS161</td></tr> <tr><th>...</th> <td>...</td> <td>...</td> <td>...</td> <td>...</td> <td>...</td> <td>...</td> <td>...</td> <td>...</td> <td>...</td> <td>...</td></tr> <tr><th>RPS10A</th> <td>0</td> <td>0</td> <td>Gene (optional)
            RPS10A                        ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[]</td> <td>49007    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>4</td> <td>0</td> <td>RPS10A</td></tr> <tr><th>SNC2</th> <td>43.5897</td> <td>5.29595</td> <td>Gene (optional)
            SNC2            Golgi apparatu...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[SNC1, SEC17, EXO84, SEC5, SEC3, SEC15, SEC6, ...</td> <td>49010    Affinity Capture-Western
            Name: Experi...</td> <td>415</td> <td>59</td> <td>17</td> <td>SNC2</td></tr> <tr><th>SVL3</th> <td>20</td> <td>3.73832</td> <td>Gene (optional)
            SVL3                cellular b...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BEM1, RPS16B, OCA6, BEM2, MNN11, MOG1, SRL3, ...</td> <td>49014    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>69</td> <td>12</td> <td>SVL3</td></tr> <tr><th>MET31</th> <td>20</td> <td>0.934579</td> <td>Gene (optional)
            MET31                         ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[STE50, RVS161, YDJ1]</td> <td>49015    Two-hybrid
            Name: Experiment_Type_(Req...</td> <td>415</td> <td>20</td> <td>3</td> <td>MET31</td></tr> <tr><th>VPS16</th> <td>35</td> <td>2.18069</td> <td>Gene (optional)
            VPS16                         ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[VAM6, SEC4, VAM7, NYV1, VPS33, YPT7, VAM3]</td> <td>49016    Co-fractionation
            Name: Experiment_Typ...</td> <td>415</td> <td>32</td> <td>7</td> <td>VPS16</td></tr> <tr><th>SGF11</th> <td>19.3103</td> <td>8.72274</td> <td>Gene (optional)
            SGF11                         ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[LTE1, RXT2, BEM1, STE50, BRE1, CYK3, SAC3, SE...</td> <td>49017    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>252</td> <td>28</td> <td>SGF11</td></tr> <tr><th>RPL21B</th> <td>12</td> <td>0.934579</td> <td>Gene (optional)
            RPL21B                        ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BEM1, PAT1, GAS1]</td> <td>49021    Positive Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>25</td> <td>3</td> <td>RPL21B</td></tr> <tr><th>ELP3</th> <td>21.608</td> <td>13.3956</td> <td>Gene (optional)
            ELP3                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[ATS1, RPS8A, RXT2, BEM1, RRP7, STE50, RVS161,...</td> <td>49022    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>295</td> <td>43</td> <td>ELP3</td></tr> <tr><th>ELP4</th> <td>24.6032</td> <td>9.65732</td> <td>Gene (optional)
            ELP4                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BEM1, STE50, RVS161, BRE1, UBP1, GET2, SWI4, ...</td> <td>49027    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>163</td> <td>31</td> <td>ELP4</td></tr> <tr><th>BEM3</th> <td>50</td> <td>4.98442</td> <td>Gene (optional)
            BEM3                  cell cor...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[CDC24, BEM1, RVS161, RVS167, BEM2, SLT2, BCK1...</td> <td>49028    Positive Genetic
            49029    Positive Ge...</td> <td>415</td> <td>40</td> <td>16</td> <td>BEM3</td></tr> <tr><th>RNY1</th> <td>25</td> <td>1.55763</td> <td>Gene (optional)
            RNY1                cytoplasm
            ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BEM1, SPT3, GIM4, RPS21B, ELP3]</td> <td>49033    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>20</td> <td>5</td> <td>RNY1</td></tr> <tr><th>KES1</th> <td>27.5229</td> <td>9.34579</td> <td>Gene (optional)
            KES1            Golgi apparatu...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BUD14, BEM1, STE50, VAM6, ARF1, SAC3, SAC7, P...</td> <td>49034    Negative Genetic
            49037    Negative Ge...</td> <td>415</td> <td>126</td> <td>30</td> <td>KES1</td></tr> <tr><th>YPL150W</th> <td>0</td> <td>0</td> <td>NaN</td> <td>NaN</td> <td>[]</td> <td>NaN</td> <td>415</td> <td>0</td> <td>0</td> <td>YPL150W</td></tr> <tr><th>RRD2</th> <td>31.0345</td> <td>2.80374</td> <td>Gene (optional)
            RRD2                         o...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[SAC3, SSD1, GIM4, CDC55, SLT2, RTT101, PBS2, ...</td> <td>49041    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>32</td> <td>9</td> <td>RRD2</td></tr> <tr><th>BEM4</th> <td>45</td> <td>2.80374</td> <td>Gene (optional)
            BEM4                    cytopl...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[CDC24, BEM1, BEM2, KSS1, CDC42, STE11, ROM2, ...</td> <td>49044    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>25</td> <td>9</td> <td>BEM4</td></tr> <tr><th>SVS1</th> <td>0</td> <td>0</td> <td>NaN</td> <td>NaN</td> <td>[]</td> <td>NaN</td> <td>415</td> <td>0</td> <td>0</td> <td>SVS1</td></tr> <tr><th>CBC2</th> <td>8.13397</td> <td>5.29595</td> <td>Gene (optional)
            CBC2            nucleus
            CBC2  ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[RPS8A, PAT1, BRE1, SAC3, SSD1, SLX9, IST3, MO...</td> <td>49047    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>369</td> <td>17</td> <td>CBC2</td></tr> <tr><th>PPQ1</th> <td>6.25</td> <td>0.311526</td> <td>Gene (optional)
            PPQ1                    cytopl...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[ELP3]</td> <td>49048    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>17</td> <td>1</td> <td>PPQ1</td></tr> <tr><th>TCO89</th> <td>20</td> <td>7.16511</td> <td>Gene (optional)
            TCO89                         ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[RPS8A, VAM6, RDI1, RPA14, SSD1, PPM1, GET2, A...</td> <td>49049    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>129</td> <td>23</td> <td>TCO89</td></tr> <tr><th>SSO1</th> <td>25</td> <td>2.49221</td> <td>Gene (optional)
            SSO1               cytoplasm
            S...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[SNC1, SEC17, ARF1, VAM7, YKT6, NYV1, SSO2, SNC2]</td> <td>49051    Affinity Capture-Western
            Name: Experi...</td> <td>415</td> <td>61</td> <td>8</td> <td>SSO1</td></tr> <tr><th>YAR1</th> <td>19.2308</td> <td>1.55763</td> <td>Gene (optional)
            YAR1                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BEM1, ELP2, IKI3, ELP6, NST1]</td> <td>49053    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>33</td> <td>5</td> <td>YAR1</td></tr> <tr><th>CLN2</th> <td>30.1724</td> <td>10.9034</td> <td>Gene (optional)
            CLN2                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[CLN3, SKT5, CHS3, BEM1, RVS161, PAT1, VAM6, C...</td> <td>49056    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>177</td> <td>35</td> <td>CLN2</td></tr> <tr><th>EAF3</th> <td>11.7647</td> <td>6.23053</td> <td>Gene (optional)
            EAF3                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[BRE1, RVS167, SPT3, VPS72, GIM4, RIM101, APQ1...</td> <td>49060    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>241</td> <td>20</td> <td>EAF3</td></tr> <tr><th>SEC8</th> <td>39.2857</td> <td>3.42679</td> <td>Gene (optional)
            SEC8                 cell cort...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[EXO84, SEC5, SEM1, RVS167, SEC3, SEC4, SEC15,...</td> <td>49062             Co-purification
            49063    Aff...</td> <td>415</td> <td>80</td> <td>11</td> <td>SEC8</td></tr> <tr><th>SPE3</th> <td>0</td> <td>0</td> <td>Gene (optional)
            SPE3                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[]</td> <td>49064    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>6</td> <td>0</td> <td>SPE3</td></tr> <tr><th>NVJ2</th> <td>0</td> <td>0</td> <td>Gene (optional)
            NVJ2                cytoplasm
            ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[]</td> <td>49065    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>2</td> <td>0</td> <td>NVJ2</td></tr> <tr><th>SYT1</th> <td>26.3158</td> <td>1.55763</td> <td>Gene (optional)
            SYT1                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[RGP1, RVS167, ISC1, RIC1, CDC42]</td> <td>49066    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>20</td> <td>5</td> <td>SYT1</td></tr> <tr><th>CLB5</th> <td>10.2362</td> <td>4.04984</td> <td>Gene (optional)
            CLB5                          ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[PAT1, MSN5, VPS72, SWI4, CDC55, RIM101, SLT2,...</td> <td>49067    Negative Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>196</td> <td>13</td> <td>CLB5</td></tr> <tr><th>RHO1</th> <td>20</td> <td>3.42679</td> <td>Gene (optional)
            RHO1                         G...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[RDI1, SAC7, SEC3, SLT2, SKN7, ROM2, ZDS1, BNI...</td> <td>49068    Co-fractionation
            Name: Experiment_Typ...</td> <td>415</td> <td>78</td> <td>11</td> <td>RHO1</td></tr> <tr><th>SKI3</th> <td>12.1951</td> <td>3.11526</td> <td>Gene (optional)
            SKI3                cytoplasm
            ...</td> <td>Gene (optional)
            BEM1                    cellul...</td> <td>[RPS8A, PAT1, SAC3, MNN10, RVS167, LSM1, RPL8B...</td> <td>49069    Positive Genetic
            Name: Experiment_Typ...</td> <td>415</td> <td>104</td> <td>10</td> <td>SKI3</td></tr></tbody></table> <p>321 rows × 10 columns</p></div> <div class="language-python extra-class"><pre class="language-python"><code>
<span class="token comment"># I dont want the query to be in the plot</span>
df<span class="token operator">=</span>pd<span class="token punctuation">.</span>DataFrame<span class="token punctuation">(</span>d2<span class="token punctuation">)</span><span class="token punctuation">.</span>T
a<span class="token operator">=</span>df
a_col<span class="token operator">=</span>a<span class="token punctuation">.</span>columns<span class="token punctuation">.</span>values
<span class="token comment">#0'% of query 2 subset  ',</span>
<span class="token comment">#1'% of query subset',</span>
<span class="token comment">#2'GO_slim_interactors'</span>
<span class="token comment">#3'GO_slim_query'</span>
<span class="token comment">#4'common',</span>
<span class="token comment">#5'interact_annotation',</span>
<span class="token comment">#6'len_i_1',</span>
<span class="token comment">#7'len_i_2',</span>
<span class="token comment">#8'n_common',</span>
<span class="token comment">#9'names of genes'</span>
ab<span class="token operator">=</span>a<span class="token punctuation">[</span><span class="token string">'% of query subset'</span><span class="token punctuation">]</span><span class="token punctuation">[</span><span class="token punctuation">:</span><span class="token punctuation">:</span><span class="token operator">-</span><span class="token number">1</span><span class="token punctuation">]</span>
pos<span class="token operator">=</span> np<span class="token punctuation">.</span>arange<span class="token punctuation">(</span><span class="token builtin">len</span><span class="token punctuation">(</span>a<span class="token punctuation">.</span>iloc<span class="token punctuation">[</span><span class="token punctuation">:</span><span class="token punctuation">,</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">)</span><span class="token punctuation">)</span>
<span class="token comment">#making a data frame of both datasets for better handling them</span>
<span class="token comment">#ab_df=pd.DataFrame({'% of query subset':ab,'names of genes':names2[::-1]})</span>
<span class="token comment">#sorting the data by the values</span>
absorted<span class="token operator">=</span>a<span class="token punctuation">.</span>sort_values<span class="token punctuation">(</span>by<span class="token operator">=</span><span class="token punctuation">[</span><span class="token string">'% of query subset'</span><span class="token punctuation">]</span><span class="token punctuation">)</span>

genes_to_plot<span class="token operator">=</span>absorted<span class="token punctuation">[</span><span class="token string">'names of genes'</span><span class="token punctuation">]</span><span class="token punctuation">.</span>iloc<span class="token punctuation">[</span><span class="token builtin">len</span><span class="token punctuation">(</span>a<span class="token punctuation">.</span>iloc<span class="token punctuation">[</span><span class="token punctuation">:</span><span class="token punctuation">,</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">)</span><span class="token operator">-</span><span class="token number">20</span><span class="token punctuation">:</span><span class="token builtin">len</span><span class="token punctuation">(</span>a<span class="token punctuation">.</span>iloc<span class="token punctuation">[</span><span class="token punctuation">:</span><span class="token punctuation">,</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">)</span><span class="token punctuation">]</span>
numbers_to_plot<span class="token operator">=</span>absorted<span class="token punctuation">[</span><span class="token string">'% of query subset'</span><span class="token punctuation">]</span><span class="token punctuation">.</span>iloc<span class="token punctuation">[</span><span class="token builtin">len</span><span class="token punctuation">(</span>a<span class="token punctuation">.</span>iloc<span class="token punctuation">[</span><span class="token punctuation">:</span><span class="token punctuation">,</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">)</span><span class="token operator">-</span><span class="token number">20</span><span class="token punctuation">:</span><span class="token builtin">len</span><span class="token punctuation">(</span>a<span class="token punctuation">.</span>iloc<span class="token punctuation">[</span><span class="token punctuation">:</span><span class="token punctuation">,</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">)</span><span class="token punctuation">]</span>

fig<span class="token punctuation">,</span> ax <span class="token operator">=</span> plt<span class="token punctuation">.</span>subplots<span class="token punctuation">(</span>figsize<span class="token operator">=</span><span class="token punctuation">(</span><span class="token number">10</span><span class="token punctuation">,</span><span class="token number">10</span><span class="token punctuation">)</span><span class="token punctuation">)</span>         <span class="token comment"># Sample figsize in inches</span>
plt<span class="token punctuation">.</span>barh<span class="token punctuation">(</span>pos<span class="token punctuation">[</span><span class="token number">0</span><span class="token punctuation">:</span><span class="token number">20</span><span class="token punctuation">]</span><span class="token punctuation">,</span>numbers_to_plot<span class="token punctuation">,</span>align<span class="token operator">=</span><span class="token string">'edge'</span><span class="token punctuation">,</span>tick_label<span class="token operator">=</span>genes_to_plot<span class="token punctuation">,</span>color<span class="token operator">=</span><span class="token punctuation">(</span><span class="token number">0.2</span><span class="token punctuation">,</span> <span class="token number">0.4</span><span class="token punctuation">,</span> <span class="token number">0.6</span><span class="token punctuation">,</span> <span class="token number">0.6</span><span class="token punctuation">)</span><span class="token punctuation">)</span>

ax<span class="token punctuation">.</span>tick_params<span class="token punctuation">(</span>labelbottom<span class="token operator">=</span><span class="token string">'on'</span><span class="token punctuation">,</span>labeltop<span class="token operator">=</span><span class="token string">'on'</span><span class="token punctuation">)</span>
ax<span class="token punctuation">.</span>grid<span class="token punctuation">(</span>color<span class="token operator">=</span><span class="token string">'k'</span><span class="token punctuation">,</span> linestyle<span class="token operator">=</span><span class="token string">'-'</span><span class="token punctuation">,</span> linewidth<span class="token operator">=</span><span class="token number">0.5</span><span class="token punctuation">)</span>

ax<span class="token punctuation">.</span>text<span class="token punctuation">(</span><span class="token number">120</span><span class="token punctuation">,</span><span class="token number">22</span><span class="token punctuation">,</span><span class="token string">&quot;Top_most_connected_gene&quot;</span><span class="token punctuation">,</span>fontsize<span class="token operator">=</span><span class="token number">17</span><span class="token punctuation">)</span>
ax<span class="token punctuation">.</span>text<span class="token punctuation">(</span><span class="token number">120</span><span class="token punctuation">,</span> <span class="token number">0</span><span class="token punctuation">,</span>  data_go<span class="token punctuation">.</span>loc<span class="token punctuation">[</span>genes_to_plot<span class="token punctuation">[</span><span class="token number">18</span><span class="token punctuation">]</span><span class="token punctuation">]</span><span class="token punctuation">[</span>col_label_go<span class="token punctuation">[</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">]</span><span class="token punctuation">,</span> fontsize<span class="token operator">=</span><span class="token number">15</span><span class="token punctuation">)</span>
ax<span class="token punctuation">.</span>text<span class="token punctuation">(</span><span class="token number">120</span><span class="token punctuation">,</span><span class="token operator">-</span><span class="token number">3</span><span class="token punctuation">,</span><span class="token string">&quot;interaction_annotation&quot;</span><span class="token punctuation">,</span>fontsize<span class="token operator">=</span><span class="token number">17</span><span class="token punctuation">)</span>
ax<span class="token punctuation">.</span>text<span class="token punctuation">(</span><span class="token number">120</span><span class="token punctuation">,</span> <span class="token operator">-</span><span class="token number">6</span><span class="token punctuation">,</span>  df<span class="token punctuation">.</span>loc<span class="token punctuation">[</span>query<span class="token punctuation">[</span><span class="token number">0</span><span class="token punctuation">]</span><span class="token punctuation">,</span>genes_to_plot<span class="token punctuation">[</span><span class="token number">18</span><span class="token punctuation">]</span><span class="token punctuation">]</span><span class="token punctuation">[</span>a_col<span class="token punctuation">[</span><span class="token number">5</span><span class="token punctuation">]</span><span class="token punctuation">]</span><span class="token punctuation">,</span> fontsize<span class="token operator">=</span><span class="token number">15</span><span class="token punctuation">)</span>
ax<span class="token punctuation">.</span>text<span class="token punctuation">(</span><span class="token number">196</span><span class="token punctuation">,</span><span class="token number">22</span><span class="token punctuation">,</span><span class="token string">&quot;Query_gene&quot;</span><span class="token punctuation">,</span>fontsize<span class="token operator">=</span><span class="token number">17</span><span class="token punctuation">)</span>
ax<span class="token punctuation">.</span>text<span class="token punctuation">(</span><span class="token number">196</span><span class="token punctuation">,</span> <span class="token number">0</span><span class="token punctuation">,</span>  data_go<span class="token punctuation">.</span>loc<span class="token punctuation">[</span>query<span class="token punctuation">]</span><span class="token punctuation">[</span>col_label_go<span class="token punctuation">[</span><span class="token number">4</span><span class="token punctuation">]</span><span class="token punctuation">]</span><span class="token punctuation">,</span> fontsize<span class="token operator">=</span><span class="token number">15</span><span class="token punctuation">)</span>
plt<span class="token punctuation">.</span>xticks<span class="token punctuation">(</span>fontsize<span class="token operator">=</span><span class="token number">18</span><span class="token punctuation">,</span> rotation<span class="token operator">=</span><span class="token number">0</span><span class="token punctuation">)</span>
plt<span class="token punctuation">.</span>yticks<span class="token punctuation">(</span>fontsize<span class="token operator">=</span><span class="token number">18</span><span class="token punctuation">,</span> rotation<span class="token operator">=</span><span class="token number">0</span><span class="token punctuation">)</span>
plt<span class="token punctuation">.</span>xlabel<span class="token punctuation">(</span><span class="token string">'Percentage of the conection of interactors with_'</span> <span class="token operator">+</span> <span class="token string">&quot;&quot;</span><span class="token punctuation">.</span>join<span class="token punctuation">(</span>query<span class="token punctuation">)</span><span class="token punctuation">,</span>fontsize<span class="token operator">=</span><span class="token number">18</span><span class="token punctuation">)</span>

plt<span class="token punctuation">.</span>savefig<span class="token punctuation">(</span><span class="token string">&quot;common_interactors_&quot;</span> <span class="token operator">+</span> <span class="token string">&quot;&quot;</span><span class="token punctuation">.</span>join<span class="token punctuation">(</span>query<span class="token punctuation">)</span> <span class="token operator">+</span> <span class="token string">&quot;.svg&quot;</span><span class="token punctuation">,</span>dpi<span class="token operator">=</span><span class="token number">300</span><span class="token punctuation">,</span><span class="token builtin">format</span><span class="token operator">=</span><span class="token string">'svg'</span><span class="token punctuation">)</span>
</code></pre></div><h2 id="results-figure"><a href="#results-figure" class="header-anchor">#</a> Results Figure</h2> <p><img src="/Experimental-journal-deploy/assets/img/output_common-genes-workflow.5cf61303.png" alt="Figure-showing-the-results"></p></div> <footer class="page-edit"><!----> <!----></footer> <div class="page-nav"><p class="inner"><span class="prev">
      ←
      <a href="/Experimental-journal-deploy/2019-07/2019-07-16-Byk832-checking-clones-ade-ura.html" class="prev">Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549 :ok:</a></span> <span class="next"><a href="/Experimental-journal-deploy/2019-07/2019-07-17_Exp-Genomic-prep-ylic133-clones.html">Title : 17072019-DNA extraction of ylic133: ura3- ade2- clones anad PCR 🆗 ✅</a>
      →
    </span></p></div> </main></div><div class="global-ui"></div></div>
    <script src="/Experimental-journal-deploy/assets/js/app.1f87a2af.js" defer></script><script src="/Experimental-journal-deploy/assets/js/2.1b349405.js" defer></script><script src="/Experimental-journal-deploy/assets/js/29.1a21047e.js" defer></script>
  </body>
</html>
